haskell/test/Stablecoin.hs

```diff
@@ -0,0 +1,94 @@
+-- SPDX-FileCopyrightText: 2020 tqtezos
+-- SPDX-License-Identifier: MIT
+
+module Stablecoin
+  ( spec_Stablecoin
+  , spec_StablecoinNettest
+  , spec_StablecoinNettestReal
+  ) where
+
+import Test.Hspec
+
+import Tezos.Core
+import Data.Maybe
+import qualified Data.Map as Map
+import Lorentz.Address as LA
+import qualified Morley.Nettest as NetTest
+import Michelson.Test.Import
+import Michelson.Test.Integrational
+import Michelson.Typed
+import qualified Michelson.Untyped as U
+import qualified Lorentz as L
+import Lorentz.Contracts.Test.FA2
+import Lorentz.Contracts.Spec.FA2Interface hiding (Parameter)
+import Lorentz.Contracts.Stablecoin as SC
+import Nettest
+
+spec_Stablecoin :: Spec
+spec_Stablecoin = specWithUntypedContract "test/resources/stablecoin.tz" specCallback
+
+spec_StablecoinNettest :: Spec
+spec_StablecoinNettest = specWithUntypedContract "test/resources/stablecoin.tz" nettest_scenario
+
+spec_StablecoinNettestReal :: Spec
+spec_StablecoinNettestReal =
+  specWithUntypedContract "test/resources/stablecoin.tz" $ real_nettest_StablecoinNettest
+
+real_nettest_StablecoinNettest :: U.Contract ->  Spec
+real_nettest_StablecoinNettest contract = describe "Stablecoin real network behavior" $
```

Comment:
We normally put it into a separate test-suite, because it requires some extra configuration (`tezos-client` that talks to a real node) and CLO parsing.

Reply:
I have disable that test for now.

Reply:
Well, in the end it should be a separate test-suite anyway because it requires custom CLO parsing. We shouldn't hardcode `MorleyClientConfig`, that's bad UX (e. g. no way to change `verbose` flag).

Probably you can try to integrate it with `tasty` so that it somehow parses its own command line options, but that seems harder than putting into a separate test-suite.